### PR TITLE
How to use Checker Framework from GitHub

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

File: .travis-build.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+fi
+if [[ "$SLUGOWNER" == "" ]]; then
+  SLUGOWNER=typetools
+fi
+echo SLUGOWNER=$SLUGOWNER
+
+echo TRAVIS_PULL_REQUEST_BRANCH = $TRAVIS_PULL_REQUEST_BRANCH
+echo TRAVIS_BRANCH = $TRAVIS_BRANCH
+
+## Build Checker Framework
```
Comment (kelloggm) [inline .travis-build.sh]: There is no reason to build the Checker Framework in this Travis build if the `-P checkerframework-local` flag is not going to be passed to maven, correct?

If so, why not guard all of this with a check against that flag?

Comment (mernst) [reply .travis-build.sh]: You are correct that building the Checker Framework is unnecessary if it won't be used, and that this wastes time in some/many jobs.

The flag can be arbitrary, so determining from the flag whether the Checker Framework is not a simple check and I didn't want to hard-code a brittle or error-prone check.

File: .travis-build.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# This script compiles Guava, performing type-checking.
```
Comment (smillst) [inline .travis-build.sh]: Before, we were using the typetest/guava-typecheck job to run the type checkers on Guava.  Are we still going to use that?

Comment (mernst) [reply .travis-build.sh]: Yes, but this permits tests on branches which that does not.

**Pull request merged**